### PR TITLE
Add Kubernetes steps for cluster restore

### PR DESCRIPTION
**Pull request by JakeSCahill**
Preview: https://deploy-preview-176--redpanda-docs-preview.netlify.app/23.3/manage/kubernetes/storage/tiered-storage/#whole-cluster-restore-for-disaster-recovery

- Added steps for cluster restore in Kubernetes.
- Restructured some of the information into clearer sections.
- Fixed instances of tiered storage in lowercase.

Fixes redpanda-data/documentation-private#2025

### CODE REVIEW

**Review by mattschumpert — APPROVED** [review]

File: modules/manage/partials/tiered-storage.adoc [hunk 1]
```diff
@@ -1313,25 +1304,157 @@ The following metadata is included in a whole cluster restore:
...
+
+* `enable_cluster_metadata_upload_loop`: Enable metadata uploads. This property is enabled by default and is required for whole cluster restore.
+* `cloud_storage_cluster_metadata_upload_interval_ms`: Set the time interval to wait between cluster metadata uploads.
+* `controller_snapshot_max_age_sec`: Maximum amount of time that can pass before Redpanda attempts to take a controller snapshot after a new controller command appears. This affects how fresh the uploaded metadata can be.
+
+ifndef::env-kubernetes[]
+==== Restore to target cluster
+
+// Update xref when rpk command alias is implemented
+You can restore data from a failed cluster to a target cluster using the xref:reference:rpk/rpk-cluster/rpk-cluster-storage-recovery.adoc[`rpk cluster storage restore`] command. The wait flag `-w` tells the command to poll the status of the restore process until completed, before exiting.
```
Comment (greg-szrama) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
You can restore data from a failed cluster to a target cluster using the xref:reference:rpk/rpk-cluster/rpk-cluster-storage-recovery.adoc[`rpk cluster storage restore`] command. The wait flag `-w` tells the command to poll the status of the restore process and then exit when completed.
```

File: modules/manage/partials/tiered-storage.adoc [hunk 2]
```diff
@@ -1313,25 +1304,157 @@ The following metadata is included in a whole cluster restore:
...
+[,bash]
+----
+rpk cluster storage restore start -w
+----
+
+==== Perform a rolling restart
+
+To ensure that the cluster reflects newly-restored configuration properties, perform a xref:manage:cluster-maintenance/rolling-restart.adoc[rolling restart] after the restore process.
+
+TIP: Run the `rpk cluster config status` command to see brokers that need to restart for updated configuration properties to take effect.
```
Comment (greg-szrama) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
TIP: Run the `rpk cluster config status` command to list the brokers that need a restart for updated configuration properties to take effect.
```

File: modules/manage/partials/tiered-storage.adoc [hunk 3]
```diff
@@ -1313,25 +1304,157 @@ The following metadata is included in a whole cluster restore:
...
 
-At this point, you may redirect your application workload to the new cluster. Make sure to update your application code to use the new cluster's addresses.
+When the cluster restore is successfully completed successfully, you can redirect your application workload to the new cluster. Make sure to update your application code to use the new addresses of your brokers.
 
 == Remote recovery
 
 When you create a topic, you can use remote recovery to download the topic data from object storage. This is useful when you need to restore a single topic that was accidentally deleted from a cluster.
 
-Note that remote recovery is only safe when no other clusters are still writing to the specified bucket or container. 
+Note that remote recovery is only safe when no other clusters are still writing to the specified bucket or container.
```
Comment (greg-szrama) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
IMPORTANT: Remote recovery is only safe when no other clusters are writing to the specified bucket or container.
```

**Review by greg-szrama — APPROVED** [review]
A few wording suggestions but otherwise looks good!

**Pull request merged**